app/src/main/java/com/brewfind/Service/BreweryService.kt

```diff
@@ -29,11 +29,11 @@ class BreweryService {
 
         })
 
-        return breweries
+        return _breweries
     }
 
     fun fetchBreweriesByArea(chosenStreet: String) : MutableLiveData<ArrayList<Brewery>> {
-        val breweries = MutableLiveData<ArrayList<Brewery>>()
+        var _breweries = MutableLiveData<ArrayList<Brewery>>()
```

Comment:
Changing val to var, allowing _breweries to be open for possible changes as well as to stay consistent with the _breweries in the internal function above.
Source: [https://www.programiz.com/kotlin-programming/variable-types](https://www.programiz.com/kotlin-programming/variable-types)

Reply:
We usually go the other direction.  A variable should be a val by default.  Only change to var if it is absolutely necessary to reassign it at some point.  If you have a variable as a var instead of a val, you can run into trouble on null checks, because you could:

1. Check for nullness and verify it's not null.
2. Reassign it to null.
3. Perform an operation that requires that it is not null.